scripts/deploy-validator.sh

```diff
@@ -114,6 +114,7 @@ PUBLIC_KEY=$(linera-server generate --validators validator-config.toml)
 echo "Validator setup completed successfully."
 echo "Starting docker compose..."
 
+export LINERA_IMAGE="us-docker.pkg.dev/linera-io-dev/linera-public-registry/linera:$BRANCH_NAME"
```

Comment:
For actual validators, should we do `release_$BRANCH_NAME`? And make sure that we tag the image with the version of the branch we want validators to use with this release tag?